controller/src/test/java/io/pravega/controller/store/stream/PravegaTablesStoreHelperTest.java

```diff
@@ -0,0 +1,183 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.store.stream;
+
+import com.google.common.collect.Lists;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import io.pravega.common.Exceptions;
+import io.pravega.common.concurrent.Futures;
+import io.pravega.controller.mocks.SegmentHelperMock;
+import io.pravega.controller.server.SegmentHelper;
+import io.pravega.controller.server.WireCommandFailedException;
+import io.pravega.controller.server.rpc.auth.AuthHelper;
+import io.pravega.shared.protocol.netty.WireCommandType;
+import io.pravega.test.common.AssertExtensions;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.spy;
+
+public class PravegaTablesStoreHelperTest {
+    private ScheduledExecutorService executor;
+    private SegmentHelper segmentHelper;
+    private AuthHelper authHelper;
+    private PravegaTablesStoreHelper storeHelper;
+
+    @Before
+    public void setup() throws Exception {
+        executor = Executors.newScheduledThreadPool(5);
+        segmentHelper = SegmentHelperMock.getSegmentHelperMockForTables(executor);
+        authHelper = AuthHelper.getDisabledAuthHelper();
+        storeHelper = new PravegaTablesStoreHelper(segmentHelper, authHelper, executor);
```

Comment:
Does it need to be closed during tear down?

Reply:
no, store helper is not closable. It has no resources that need to be closed.